addons/l10n_hr/wizard/account_move_send_batch_wizard.py

```diff
@@ -0,0 +1,15 @@
+import pytz
+
+from odoo import fields, models
+
+
+class AccountMoveSendBatchWizard(models.TransientModel):
+    _inherit = 'account.move.send.batch.wizard'
+
+    def action_send_and_print(self, force_synchronous=False, allow_fallback_pdf=False):
+        # EXTENDS 'account'
+        self.ensure_one()
+        for move in self.move_ids:
+            if move.company_id.country_id.code == 'HR' and move.move_type in ['out_invoice', 'out_refund']:
```

Comment:
```suggestion
            if move.country_code == 'HR' and move.is_sale_document():
```

This uses the company's `account_fiscal_country_id` (fiscal country) which would be the appropriate one here